apps/web/src/pages/index.tsx

```diff
@@ -14,13 +16,29 @@ interface HomeProps {
 
 export default function Home({ page }: HomeProps) {
   const { push } = useRouter()
+  const auth = useAuth()
 
   const onClickFeatured = useCallback(() => {
     if (page.featuredPack) {
       push(urls.release.replace(':packSlug', page.featuredPack.slug))
     }
   }, [page.featuredPack, push])
 
+  useEffect(() => {
+    const findUser = async () => {
+      try {
+        const claims = await adminService.getLoggedInUserPermissions()
+        console.log('claims:', claims)
+      } catch (error) {
+        console.error(error)
+      }
+    }
+    // Check permissions on page render, after auth token is refreshed so claims are fresh
+    if (auth.user) {
+      findUser()
+    }
+  }, [auth?.user])
```

Comment:
If user is an object you may want to have this depend on something like `auth?.user.email`